recipes/kalpy/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "kalpy" %}
+{% set version = "0.0.4" %}
+
+package:
+  name: kalpy
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/k/kalpy-kaldi/kalpy-kaldi-{{ version }}.tar.gz
+  sha256: 43cf28d35a97751d4f1b40f133897c67f66dfd81bb0d8d010c826998cca2b49e
```

Comment:
Can you explain the situation a little. On the README of , I see:
> Kalpy is also available on pip via the mfa-kaldi package

what's the relationship between kalpy and mfa-kaldi? I guess kalpy is the more fully-featured one (includes the lib)? Should we add an output `mfa-kaldi` here that just wraps `kalpy` to ensure that no-one gets the idea of packaging `mfa-kalpi` separately?

Reply:
Oh this is just a typo left over from a previous one.  Just plain `kalpy` is too close to some other package on pypi, I think `kaipy` is the conflicting one.  So I was struggling with what to name it, but at the moment it's `kalpy-kaldi` which should help with discoverability considering the number of other binding libraries on pip at various stages of abandonment.  In this case, should the conda recipe output both `kalpy` and `kalpy-kaldi`?  Should I figure out another name that allows for more transparency and consistency across both systems?

I'm also on the fence about having it on pypi to begin with, because it does basically require having the shared libraries from the kaldi conda package installed anyway.

The three options I was considering were:

1. (Current set up) Have just sdist on Pypi, installation requires having kaldi shared libraries where KALDI_ROOT is pointing.  Upside is that I can list kalpy in the dependencies for montreal-forced-aligner, but pip installation is not as easy or fully packaged as I would like.  I haven't really looked into the work required to get wheels working for each platform that would not depend on the conda package, but that would simplify pip installation.
2. No package on pypi, just a conda package, which would make more sense in my mind because having a pip package depend on conda packages seems like a broken experience, also doesn't get the autodetection of new versions.
3. Have kalpy be an output of the kaldi feedstock, since they are largely just bindings for the kaldi libraries, but I think this option probably complicates that feedstock too much.

Do you have any thoughts on best practices here?

Reply:
> Do you have any thoughts on best practices here?

I can only really comment on conda-forge, distribution outside of that is up to the respective maintainers. I agree with much of your reasoning (pypi packages depending on conda-forge makes no sense, "better" [for PyPI] would be vendoring libkaldi in the kalpy wheel), but it doesn't really matter for our purposes here.

Also, IIUC `mfa-kaldi` in the readme was a leftover from something else (it's not there anymore in the repo), so that sounds fine as well.